packages/dom-helper/tests/prop-test.js

```diff
@@ -3,17 +3,18 @@ import { normalizeProperty } from 'dom-helper/prop';
 QUnit.module('dom-helper prop');
 
 test('type.attr, for element props that for one reason or another need to be treated as attrs', function() {
-  expect(13);
+  expect(14);
 
   [
+    { tagName: 'A',        key: 'download' },
     { tagName: 'TEXTAREA', key: 'form' },
     { tagName: 'BUTTON',   key: 'type' },
     { tagName: 'INPUT',    key: 'type' },
     { tagName: 'INPUT',    key: 'list' },
     { tagName: 'INPUT',    key: 'form' },
     { tagName: 'INPUT',    key: 'autocorrect' },
     { tagName: 'OPTION',   key: 'form' },
-    { tagName: 'INPUT',    key: 'form' },
+    { tagName: 'SELECT',   key: 'form' },
```

Comment:
`{ tagName: 'INPUT',    key: 'form' }` was duplicated here and `{ tagName: 'SELECT',   key: 'form' }` was not present but should have been. Unrelated to the issue but I figured I'd fix it while I was in here.